src/dashboard/cd-dashboard-stats.vue

```diff
@@ -1,16 +1,49 @@
 <template>
   <div class="column">
     <div class="cd-dashboard-stats">
-      <h1 class="cd-dashboard-stats__header">Dojo Stats</h1>
+      <h1 class="cd-dashboard-stats__header">{{ $t('Dojo Stats') }}</h1>
+      <div class="cd-dashboard-stats__number">
+        <div class="cd-dashboard-stats__number-value"><span>{{ bookedChildren }}</span></div>
+        <div class="cd-dashboard-stats__description">{{ $t('kids attended your events') }}</div>
```

Comment:
I have no idea how a stat dashboard looks like in different languages

Reply:
I guess it needs to be phrased in a way you could put a colon at the end, then put the number. So, like "kids attended your events: 20" doesn't work, but "kids who have attended your events: 20" does (although is now too long).

Also, since when do we use the word "kids" in the UI?